src/core/Expander/Expander.md

```diff
@@ -33,54 +33,37 @@ const [expanderThreeOpen, setExpanderThreeOpen] = React.useState(
 );
 
 <>
-  <div
-    style={{
-      display: 'flex',
-      flexDirection: 'column',
-      justifyContent: 'space-between'
-    }}
-  >
-    <div
-      style={{
-        width: 500,
-        display: 'flex',
-        flexDirection: 'row',
-        justifyContent: 'space-between'
-      }}
-    >
-      <Button onClick={() => setExpanderOneOpen(!expanderOneOpen)}>
-        Toggle Expander 1
-      </Button>
-      <Button onClick={() => setExpanderTwoOpen(!expanderTwoOpen)}>
-        Toggle Expander 2
-      </Button>
-      <Button
-        onClick={() => setExpanderThreeOpen(!expanderThreeOpen)}
-      >
-        Toggle Expander 3
-      </Button>
-    </div>
-  </div>
-
   <Expander.group OpenAll="Open all" CloseAll="Close all">
     <Expander
       title="Test expander 1"
       open={expanderOneOpen}
-      onClick={() => setExpanderOneOpen(!expanderOneOpen)}
+      onClick={() => {
```

Comment:
I'd propose to have the first two to use uncontrolled state and leave the last one as it is now? This was a good test, but I think it should be sufficient to have just one confirm?

Reply:
This is now changed as suggested; I removed the controlled state of the first two Expanders and left it only for the third one.